WORKSPACE

```diff
@@ -0,0 +1,22 @@
+workspace(name="maliput")
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "fmt",
+    urls = ["https://github.com/fmtlib/fmt/releases/download/9.1.0/fmt-9.1.0.zip"],
+    strip_prefix = "fmt-9.1.0",
+    build_file = "@//external:fmt.BUILD"
+)
+
+http_archive(
+    name = "yaml-cpp",
+    urls = ["https://github.com/jbeder/yaml-cpp/archive/refs/tags/yaml-cpp-0.7.0.zip"],
+    strip_prefix = "yaml-cpp-yaml-cpp-0.7.0"
+)
+
+http_archive(
+    name = "com_google_googletest",
+    urls = ["https://github.com/google/googletest/archive/refs/tags/release-1.12.1.zip"],
+    strip_prefix = "googletest-release-1.12.1",
```

Comment:
> When working with Bazel I guess that relying on rosdep for installing the dependencies of the package isn't expected. That's why dependencies should be brought via bazel's commands. Is that correct @hidmic ? I am not that familiar with Bazel workspaces.

Bringing dependencies via `bazel` repository rules is the standard way, but not the only one really. We could, for instance, use `rosdep` to install dependencies and then use a [`new_local_repository`](https://bazel.build/reference/be/workspace#new_local_repository) rules to scrape headers, libraries, and the like from the system. But it's a bit more brittle. 

> In order to guarantee the correct and similar functioning probably makes sense to match the version of the dependencies as if they are being installed via rosdep.

> yaml-cpp -> 0.6.2
> fmt ->6.1.2

Agreed. @erasmomontes mind to update versions?

---

FWIW there are some kinks when using `bazel` and ROS 2 that have not been fully fleshed out. Since `maliput` is ROS 2 agnostic, I wouldn't mind that much about workflow compatibility just yet.

Reply:
> Agreed. @erasmomontes mind to update versions?
Sure!